e2e/discussions.spec.ts

```diff
@@ -0,0 +1,465 @@
+import { Page, expect, test } from '@playwright/test'
+import { testAuth } from './fixtures/authenticatedUserPages'
+import { entityUrlPathname } from './helpers/entities'
+import {
+  setupProjectWithPermissions,
+  teardownProjectsAndFileHandles,
+} from './helpers/setupTeardown'
+import { dismissAlert } from './helpers/testUser'
+import { Project } from './helpers/types'
+import { UserPrefixes, userConfigs } from './helpers/userConfig'
+
+const discussionActionIconClasses = {
+  PIN: '.fa-thumb-tack.imageButton',
+  EDIT: '.fa-pencil.imageButton',
+  DELETE: '.syn-trash-o.imageButton',
+  TOGGLE_FOLLOW: '.fa-eye.imageButton',
+  LINK: '.fa-link.imageButton',
+}
+const discussionThreadSelector = '.discussionThread:visible'
+const discussionReplySelector = '.discussionReply:visible'
```

Comment:
Used to select the current thread (visible) instead of the deleted thread (hidden)